cvprac/cvp_api.py

```diff
@@ -249,6 +249,87 @@ def get_inventory(self, start=0, end=0, query=''):
                              (query, start, end), timeout=self.request_timeout)
         return data['netElementList']
 
+    def add_device_to_inventory(self, device_ip, parent_name, parent_key):
+        ''' Add the container to the specified parent.
+
+            Args:
+                device_ip (str): ip address of device we are adding
+                parent_name (str): Parent container name
+                parent_key (str): Parent container key
+        '''
+        self.log.debug('add_device_to_inventory: called')
+        data = {'data': [
+                        {
+                             'containerName' : parent_name,
+                             'containerId' : parent_key,
+                             'containerType' : 'Existing',
+                             'ipAddress' : device_ip,
+                             'containerList' : []
+                        }
+                        ]
+                }
+        data = self.clnt.post('/inventory/add/addToInventory.do?'
+                             'startIndex=0&endIndex=0',
+                             data=data,
+                             timeout=self.request_timeout)
+
+    def retry_add_to_inventory(self, device_mac, device_ip, username,
+                            password):
+        '''Retry addition of device to Cvp inventory
+
+            Args:
+                device_mac (str): MAC address of device
+                device_ip (str): ip address assigned to device
+                username (str): username for device login
+                password (str): password for user
+        '''
+        self.log.debug('retry_add_to_inventory: called')
+        data = { "key" : device_mac,
+                 "ipAddress" : device_ip,
+                 "userName" : username,
+                 "password" : password
+               }
+        data = self.clnt.post('/inventory/add/retryAddDeviceToInventory.do?'
```

Comment:
Is intention to return something from or all of data?

Reply:
Should return nothing. I'll fix.